packages/fhir-converter/src/templates/cda/Sections/Results.hbs

```diff
@@ -30,8 +30,15 @@
                                 {{>Resources/Practitioner.hbs practitioner=obsEntry.observation.author.assignedAuthor ID=practitionerId.Id}},
                                 {{>References/Observation/performer.hbs ID=(generateUUID (toJsonString obsEntry.observation)) REF=(concat 'Practitioner/' practitionerId.Id )}},                           
                             {{/with}}
+                        {{!-- else should use author from top level --}}
```

Comment:
these observation resources are nested within a results section which has a top level practitioner we extract from performer.assignedEntity (but maybe it should be author.assignedAuthor). So if there is no performer.assignedEntity specified inside the observation, lets take the observation from the top level.

Reply:
Wrt whether to use performer.assignedEntity or author.AssignedAuthor. In the xmls I was seeing, author.AssignedAuthor was generally richer. Throughout the templates, we use:
- author.AssignedAuthor: 13 times to generate practitionerId
- assignedEntity: 7 times to generate practitionerId

Not sure what to make of this. Requires more understanding